.github/workflows/site.yaml

```diff
@@ -0,0 +1,30 @@
+name: Site
+on:
+  - pull_request
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+jobs:
+  preview:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.x
+          check-latest: true
+          cache: 'pip'
+      - uses: quarto-dev/quarto-actions/setup@v2
+      - run: make deps
+      - run: make dev
+      - run: make docs
+      - uses: actions/setup-node@v4
+      - id: netlify
+        env:
+            NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+        run: |
+          preview_url=$(make --silent -C ./docs netlify | tail -n 1 | tr -d '"')
+          echo "# 🚀 Site Preview" >> $GITHUB_STEP_SUMMARY
```

Comment:
This just prints in the CI logs right?

Reply:
Renders in the preview summary: https://github.com/posit-dev/posit-sdk-py/actions/runs/9452439576?pr=214

(bottom of page) \

Reply:
Oh nice, that's not bad

Reply:
I started playing with writing a comment to the PR, but updating existing comments on re-runs ended up taking more effort than it was worth.